lib/virt_autotest/utils.pm

```diff
@@ -224,7 +244,8 @@ sub ssh_copy_id {
     my $authorized_keys = $args{authorized_keys} // '.ssh/authorized_keys';
     my $scp = $args{scp} // 0;
     my $mode = is_sle('=11-sp4') ? '' : '-f';
-    my $default_ssh_key = (!(get_var('VIRT_AUTOTEST'))) ? "/root/.ssh/id_rsa.pub" : "/var/testvirt.net/.ssh/id_rsa.pub";
+    my $default_ssh_key = $args{default_ssh_key};
+    $default_ssh_key //= (!(get_var('VIRT_AUTOTEST'))) ? "/root/.ssh/id_rsa.pub" : "/var/testvirt.net/.ssh/id_rsa.pub";
```

Comment:
Should be
```
$args{default_ssh_key} //= (!(get_var('VIRT_AUTOTEST'))) ? "/root/.ssh/id_rsa.pub" : "/var/testvirt.net/.ssh/id_rsa.pub";
```
Better ?

Reply:
I do not think args should be changed, which is passed in.